velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -60,6 +60,17 @@ class HiveConnectorTestBase : public OperatorTestBase {
       const std::function<std::unique_ptr<dwrf::DWRFFlushPolicy>()>&
           flushPolicyFactory = nullptr);
 
```

Comment:
This file is still dependent on #include "velox/dwio/dwrf/writer/FlushPolicy.h"
Remove the dependency of it.

Reply:
Removing `#include "velox/dwio/dwrf/writer/FlushPolicy.h"` in HiveConnectorTestBase.h requires me to add it to IcebergReadTest.cpp, is that ok?